packages/components/src/components/ScalarColorModeToggle/ScalarColorModeToggleIcon.vue

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+import type { Component } from 'vue'
+
+import { useBindCx } from '../../hooks/useBindCx'
+import type ScalarColorModeToggle from './ScalarColorModeToggle.vue'
+
+/**
+ * Scalar Color Mode Toggle Icon component
+ *
+ * An icon that toggles between a sun and moon.
+ *
+ * If you need a toggle that is aware of and sets color mode,
+ * you want {@link ScalarColorModeToggle} instead.
+ *
+ * @example
+ *   <ScalarColorModeToggleIcon v-model="isDarkMode" />
+ */
+export default {}
+</script>
+<script lang="ts" setup>
+const { is = 'button', mode = 'light' } = defineProps<{
+  /** The element or component this component should render as. */
+  is?: string | Component
+  /** Whether the toggle is a sun or moon icon */
+  mode?: 'light' | 'dark'
+}>()
+
+defineOptions({ inheritAttrs: false })
+const { cx } = useBindCx()
+</script>
+<template>
+  <!-- Icon -->
+  <component
+    :is="is"
+    :class="`toggle-icon-${mode}`"
+    :type="is === 'button' ? 'button' : undefined"
+    v-bind="cx('size-3 flex items-center justify-center text-c-1')">
+    <span class="toggle-icon-sun-ray" />
+    <span class="toggle-icon-sun-ray" />
+    <span class="toggle-icon-sun-ray" />
+    <span class="toggle-icon-sun-ray" />
+    <span class="toggle-icon-ellipse">
+      <span class="toggle-icon-moon-mask" />
+    </span>
+  </component>
+</template>
+<style scoped>
```

Comment:
I tried to rewrite these styles as both tailwind and an svg animation and eventually gave up

Reply:
this is the way, when its complex just go to css